src/components/BottomBar/SkyBrowser/WorldWideTelescope.jsx

```diff
@@ -0,0 +1,226 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import Picker from '../Picker';
+import FloatingWindow from './WindowThreeStates/FloatingWindow'
+import styles from './WorldWideTelescope.scss'
+import { SkyBrowser_ShowTitleInBrowserKey, SkyBrowser_InverseZoomDirectionKey } from '../../../api/keys';
+import { getBoolPropertyValue } from '../../../utils/propertyTreeHelpers';
+
+class WorldWideTelescope extends Component {
+  constructor(props) {
+    super(props);
+    this.iframe = React.createRef();
+    this.state = {
+      isDragging: false,
+      startDragPosition: [0,0],
+      topBarHeight: 25,
+    };
+    this.sendMessageToWwt = this.sendMessageToWwt.bind(this);
+    this.setAim = this.setAim.bind(this);
+    this.mouseDown = this.mouseDown.bind(this);
+    this.mouseUp = this.mouseUp.bind(this);
+    this.handleDrag = this.handleDrag.bind(this);
+    this.handleCallbackMessage = this.handleCallbackMessage.bind(this);
+    this.addAllSelectedImages = this.addAllSelectedImages.bind(this);
+    this.scroll = this.scroll.bind(this);
+    this.changeSize = this.changeSize.bind(this);
+    this.setBorderColor = this.setBorderColor.bind(this);
+    this.color = [255, 255, 255];
+    this.ratio = 1;
+  }
+
+  componentDidMount() {
+    this.props.setMessageFunction(this.sendMessageToWwt);
+    window.addEventListener("message", this.handleCallbackMessage);
+    this.props.setImageCollectionIsLoaded(false);
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener("message", this.handleCallbackMessage);
+  }
+
+  handleCallbackMessage(event) {
+    if (event.data == "wwt_has_loaded") {
+      this.sendMessageToWwt({
+        event : "modify_settings",
+        settings : [["hideAllChrome", true]],
+        target: "app"
+      });
+      this.sendMessageToWwt({
+        event: "load_image_collection",
+        url:"https://data.openspaceproject.com/wwt/1/imagecollection.wtml",
+        loadChildFolders: true
+      });
+      if (this.props.browser) {
+        this.setBorderColor(this.props.browser.color);
+      }
+    }
+    if (event.data == "load_image_collection_completed") {
+      this.props.setImageCollectionIsLoaded(true);
+      this.addAllSelectedImages();
+    }
+  }
+
+  setBorderColor(color) {
+    this.color = color;
+    this.sendMessageToWwt({
+      event: "set_background_color",
+      data: color}
+    );
+  }
+
+  addAllSelectedImages() {
+    this.props.selectedImages.reverse().map(image => {
+      this.props.selectImage(image.identifier, false);
+    });
+  }
+
+  sendMessageToWwt(message) {
+    try {
+      var frame = this.iframe.current.contentWindow;
```

Comment:
Does it make a difference whether it is `var` or `let`?

Reply:
No - I'll change it